jetty-core/jetty-security/src/main/java/org/eclipse/jetty/security/SecurityHandler.java

```diff
@@ -730,6 +730,9 @@ public SecurityHandler getSecurityHandler()
      *   <li>{@code "/admin/super/index.html"} matches {@code "/*"}, {@code "/admin/*"} and {@code "/admin/super/*"}, resulting in a
      *       constraint of {@link Authorization#SPECIFIC_ROLE} and {@link Transport#SECURE}.</li>
      * </ul>
+     * <p>If a request path is uncovered, that is there is no match for the request path, then the constraint is
```

Comment:
I'm not sure "uncovered" is the right word to use here as it is confusing with servlet uncovered methods, which is when it was covered by a less specific path spec, but the more specific path spec does not include the method.

I'n this case it is more "unmatched" rather than the "uncovered".